CHANGELOG.md

```diff
@@ -6221,6 +6228,7 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 [v11.1.1]: https://github.com/nasa/cumulus/compare/v11.1.0...v11.1.1
 [v11.1.0]: https://github.com/nasa/cumulus/compare/v11.0.0...v11.1.0
 [v11.0.0]: https://github.com/nasa/cumulus/compare/v10.1.2...v11.0.0
```

Comment:
I think this needs to be `v10.1.3...v11.0.0`

Reply:
Ah nice catch!! Thanks for that, I've updated it https://github.com/nasa/cumulus/pull/3039/commits/aaea620f8fafc3fbe6f699e36dcd4a7c7b0e21e5